src/widgets/LineChart.js

```diff
@@ -36,10 +35,14 @@ export const LineChart = ({ width, height, data }) => {
   };
   return (
     <VictoryChart width={width} height={height} padding={padding}>
-      <VictoryScatter size={dotSize} style={dotStyle} data={values} />
-      <VictoryLine style={lineStyle} data={values} />
-      {renderXAxis()}
+      {data.map(({ values }) => (
+        <VictoryScatter size={dotSize} style={dotStyle} data={values} />
+      ))}
```

Comment:
You can use hooks here to memoize:

`const scatterLine = React.useMemo(() => data.map...., [data])`

then below.. `{scatterLine}`

Although not 100% needed, good to know

Reply:
Nice! I'll give it a try later, thanks for sharing this